sdk/Lusid.Sdk.Tests/tutorials/Instruments/ContractForDifferencesExamples.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class ContractForDifferencesExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope,ModelSelection.ModelEnum model, LusidInstrument cfd)
+        {
+            var equityRequest = TestDataUtilities.BuildEquityQuoteRequest(
+                "some-id",
+                new DateTimeOffset(2019, 2, 7, 0, 0, 0, TimeSpan.Zero),
```

Comment:
could we label the variables e.g. `var x = new Thing(startDate : ... , maturity: ....)`